cli/jobs/single-step/gpu_perf/README.md

```diff
@@ -0,0 +1,126 @@
+---
+page_type: sample
+languages:
+- azurecli
+- shell
+- docker
+products:
+- azure-machine-learning
+description: This sample shows how to run standard health checks on a GPU cluster to test performance and correctness of distributed multinode GPU jobs. It helps with troubleshooting performance issues related to the environment and container.
+---
+
+# Run NCCL tests on GPU to check performance and configuration
```

Comment:
in general this kinda lends itself to a "how-to" doc -- why not host the code snippets in here and write a doc there?

Reply:
Can you point me in the right direction on where to write how-tos ?

Reply:
done internally